compute/compute-node.Dockerfile

```diff
@@ -19,7 +19,7 @@ ARG DEBIAN_VERSION
 SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
 
 RUN echo 'Acquire::Retries "5";' > /etc/apt/apt.conf.d/80-retries && \
-    echo -e "retry_connrefused = on\ntimeout=15\ntries=5\n" > /root/.wgetrc \
+    echo -e "retry_connrefused = on\ntimeout=15\ntries=5\n" > /root/.wgetrc && \
```

Comment:
either drop `-e` here as well, or replace `echo` with `/usr/bin/echo`
as base image is `debian`